civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -10,33 +10,88 @@ use Drupal\civihr_employee_portal\Helpers\HelperClass;
+      [
+        'module_name' => 'date_popup',
+        'file_name' => '/themes/datepicker.1.7.css',
+      ],
+      ['module_name' => 'panels', 'file_name' => '/css/panels.css'],
+      ['module_name' => 'radix_layouts', 'file_name' => '/radix_layouts.css'],
+      ['module_name' => 'views_tooltip', 'file_name' => '/views_tooltip.css'],
+      ['module_name' => 'tipsy', 'file_name' => '/stylesheets/tipsy.css'],
+    ];
+    // Removes above files.
+    _remove_alter_files_from_civicrm($css, $removeFiles);
+  }
 }
 
 /**
  * Implements hook_js_alter()
  */
 function civihr_employee_portal_js_alter(&$javascript) {
 
-    // Unset jcarousel on panel admin pages as it's causes infinite loops
-    // "admin/structure/pages"
-    if (strpos(current_path(), 'admin/structure/pages') !== false) {
-        foreach ($javascript['settings']['data'] as $key => $value) {
-            if (isset($value['jcarousel'])) {
-
-                unset($javascript['settings']['data'][$key]);
-            }
-        }
-
+  // Unset jcarousel on panel admin pages as it's causes infinite loops
+  // "admin/structure/pages"
+  if (strpos(current_path(), 'admin/structure/pages') !== FALSE) {
+    foreach ($javascript['settings']['data'] as $key => $value) {
+      if (isset($value['jcarousel'])) {
+        unset($javascript['settings']['data'][$key]);
+      }
     }
+  }
 
-    global $user;
-    $uf = get_civihr_uf_match_data($user->uid);
-    drupal_add_js(array('currentDrupalUserId' => $user->uid), 'setting');
-    drupal_add_js(array('currentCiviCRMUserId' => $uf['contact_id']), 'setting');
+  global $user;
+  $uf = get_civihr_uf_match_data($user->uid);
+  drupal_add_js(['currentDrupalUserId' => $user->uid], 'setting');
+  drupal_add_js(['currentCiviCRMUserId' => $uf['contact_id']], 'setting');
+
+  _setup_modals();
+
+  // Removing unwanted js files from civicrm pages.
+  if (civicrm_invoke()) {
+
+    // Multi-dimensional array containing module name and file path.
+    $removeFiles = [
+      ['module_name' => 'tipsy', 'file_name' => '/javascripts/jquery.tipsy.js'],
+      ['module_name' => 'tipsy', 'file_name' => '/javascripts/tipsy.js'],
+      [
+        'module_name' => 'jquery_update',
+        'file_name' => '/replace/ui/external/jquery.cookie.js',
+      ],
+      [
+        'module_name' => 'jquery_update',
+        'file_name' => '/replace/jquery/1.8/jquery.min.js',
+      ],
+    ];
+    // Removes above files.
+    _remove_alter_files_from_civicrm($javascript, $removeFiles);
+  }
+}
 
-    _setup_modals();
+/**
+ * Removes javascript/css from civicrm pages.
+ *
+ * @param array &$style_type variable contains all the references to loaded
+ *   js/css array from alter.
+ *
+ * @param array $removeFiles contains multi-dimensional array : module_name key
+ *   as name of the contrib module from which we need to remove corresponding
+ *   file using file_name key.
+ */
+function _remove_alter_files_from_civicrm(&$style_type, $removeFiles = []) {
```

Comment:
I don't think neither the function nor the param names are good enough here, especially `$style_type`, which sounds it would be a string (either "css" or "js"). What about

```php
function _remove_resources($resourcesList, $blacklist = []) {

}
```
the name of the function now is more generic and more "reusable"